src/main/java/gov/cms/mat/cql_elm_translation/controllers/CqlToolsController.java

```diff
@@ -92,15 +91,11 @@ public ResponseEntity<Map<String, Set<CQLDefinition>>> getDefinitionCallstack(
   }
 
   @PutMapping(
-      value = "/cql/lookups",
+      value = "/cql-builder-lookups",
       produces = MediaType.APPLICATION_JSON_VALUE,
       consumes = MediaType.APPLICATION_JSON_VALUE)
-  public ResponseEntity<CqlLookups> getCqlLookups(
-      @RequestBody CqlLookupRequest lookupRequest,
-      @RequestHeader("Authorization") String accessToken) {
-    log.info("Translator is preparing CQL Lookups for simple xml");
-    return ResponseEntity.ok(
-        cqlParsingService.getCqlLookups(
-            lookupRequest.getCql(), lookupRequest.getMeasureExpressions(), accessToken));
+  public ResponseEntity<CqlBuilderLookup> getCqlBuilderLookups(
```

Comment:
Would this affect UI since we change the return type?

Reply:
it should not because this endpoint was QDM specific which is no longer needed in FHIR service. So I replaced it with this one